portfolio/src/main/webapp/index.html

```diff
@@ -23,24 +23,23 @@ <h1>Tal's Portfolio</h1>
       <p>Click here to get a random fact about me!:</p>
       <button id = "button" onclick="addRandomFact()">About Me</button>
       <div id="fact-container"></div>
- 
-      <p>Post a comment here!</p>
+       
       
+     
       
-      <div id = 'notLoggedIn'>
-          <p>Login To Comment</p>
-          <button target="loginUrl">Login</button>
-      </div>
+      <form id = 'notLoggedIn' method="GET" action="loginUrl">
```

Comment:
When I do action="/user" it works but I want it to go directly to the loginUrl from the UserServlet.java. Is there a way to go there directly? Right now I get a 404 error.